cmd/bsky-webhook/main.go

```diff
@@ -21,137 +22,102 @@ import (
 	"github.com/klauspost/compress/zstd"
 )
 
-type BskyMessage struct {
-	Did    string      `json:"did"`
-	Commit *BskyCommit `json:"commit"`
-	Kind   string      `json:"kind"`
-}
-
-func (m *BskyMessage) toURL(handle *string) string {
-	author := handle
-	if author == nil {
-		author = &m.Did
-	}
-
-	return fmt.Sprintf("https://bsky.app/profile/%s/post/%s", url.PathEscape(*author), url.PathEscape(m.Commit.Rkey))
-}
-
-type BskyCommit struct {
-	Rev       string      `json:"rev"`
-	Rkey      string      `json:"rkey"`
-	Record    *BskyRecord `json:"record"`
-	Operation string      `json:"operation"`
-}
-
-type BskyRecord struct {
-	Text  string    `json:"text"`
-	Embed BskyEmbed `json:"embed"`
-}
-
-type BskyEmbed struct {
-	Images []BskyImage `json:"images"`
-}
-
-type BskyImage struct {
-	Image BskyInnerImage `json:"image"`
-}
-
-type BskyInnerImage struct {
-	Ref BskyImageRef `json:"ref"`
-}
-
-type BskyImageRef struct {
-	Link string `json:"$link"`
-}
-
-type SlackAttachment struct {
-	AuthorName string `json:"author_name"`
-	AuthorIcon string `json:"author_icon"`
-	AuthorLink string `json:"author_link"`
-	Text       string `json:"text"`
-	ImageUrl   string `json:"image_url"`
-	Footer     string `json:"footer"`
-}
-
-type SlackBody struct {
-	Text        string            `json:"text"`
-	UnfurlLinks bool              `json:"unfurl_links"`
-	UnfurlMedia bool              `json:"unfurl_media"`
-	Attachments []SlackAttachment `json:"attachments"`
-}
-
-func envOr(key, defaultVal string) string {
-	if result, ok := os.LookupEnv(key); ok {
-		return result
-	}
-	return defaultVal
-}
-
-var addr = flag.String("addr", envOr("JETSTREAM_ADDRESS", ""), "jetstream websocket address")
-var bskyHandle = flag.String("bskyHandle", envOr("BSKY_HANDLE", ""), "bluesky handle for auth")
-var bskyAppKey = flag.String("bskyAppPassword", envOr("BSKY_APP_PASSWORD", ""), "bluesky app password for auth")
-var webhookUrl = flag.String("slackWebhookUrl", envOr("SLACK_WEBHOOK_URL", ""), "slack webhook url")
-var bskyServerUrl = flag.String("bskyServerUrl", envOr("BSKY_SERVER_URL", "https://bsky.network"), "bluesky pds server url")
-var watchWord = flag.String("watchWord", envOr("WATCH_WORD", "tailscale"), "the word to watch out for. may be multiple words in futureee")
+var (
+	addr = flag.String("addr", envOr("JETSTREAM_ADDRESS", ""),
+		"jetstream websocket address")
+	bskyHandle = flag.String("bsky-handle", envOr("BSKY_HANDLE", ""),
+		"bluesky handle for auth (required)")
+	bskyAppKey = flag.String("bsky-app-password", envOr("BSKY_APP_PASSWORD", ""),
+		"bluesky app password for auth (required)")
+	webhookURL = flag.String("slack-webhook-url", envOr("SLACK_WEBHOOK_URL", ""),
+		"slack webhook URL (required)")
+	bskyServerURL = flag.String("bsky-server-url", envOr("BSKY_SERVER_URL",
+		"https://bsky.network"), "bluesky pds server URL (required)")
```

Comment:
Is this only marked required because of the default? 
Using `https://bsky.network` as a default will work for ~99% of users. You only need to specify one if your account is on your own Bluesky PDS (Personal Data Server), so users aren't required to set this.